src/libs/DeviceCapabilities/hasHoverSupport/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Allows us to identify whether the platform is hoverable.
+ *
+ * @returns {Boolean}
+ */
+
+import * as Browser from '../../Browser';
+
+function hasHoverSupport() {
+    // First check internet explorer, because it doesn't support interaction media queries:
+    // https://caniuse.com/?search=media%20features
+    if (Browser.isInternetExplorer()) {
+        return !Browser.isMobile();
+    }
+    return !window.matchMedia('(hover: none)').matches;
```

Comment:
Isn't this enough to detect that IE doesn't have hover support? I tested it in IE11 and it appears to work. I'd like to get rid of `Browser.isInternetExplorer()` if at all possible.

Reply:
I don't have access to a windows machine so I'm not sure how to check IE. I was just trusting caniuse: https://caniuse.com/?search=media%20features

> Isn't this enough to detect that IE doesn't have hover support?

IE doesn't have hover support? It might just be that `matchMedia` is returning no matches because media queries aren't supported.

> I'd like to get rid of Browser.isInternetExplorer() if at all possible.

I'm fine with getting rid of this regardless. I don't really think our target users will still be using IE and it's officially EOL as of Summer 2022.

Reply:
Removed `Browser.isInternetExplorer`

Reply:
FYI, you can use browserstack. We have an account in 1Password for it.